assets/js/modules/analytics/components/settings/SettingsControls.js

```diff
@@ -16,33 +16,66 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { MODULES_ANALYTICS, PROFILE_CREATE } from '../../datastore/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import {
 	AccountSelect,
 	ProfileNameTextField,
 	ProfileSelect,
 	PropertySelect,
 } from '../common';
 import SettingsUseSnippetSwitch from './SettingsUseSnippetSwitch';
+import SettingsNotice from '../../../../components/SettingsNotice/SettingsNotice';
+import { TYPE_INFO } from '../../../../components/SettingsNotice';
+import WarningIcon from '../../../../../../assets/svg/icons/warning-icon.svg';
 const { useSelect } = Data;
 
-export default function SettingsControls() {
+export default function SettingsControls( { hasModuleAccess = true } ) {
 	const profileID = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getProfileID()
 	);
 
+	const module = useSelect( ( select ) =>
+		select( CORE_MODULES ).getModule( 'analytics' )
+	);
+
+	const ownerName = module?.owner?.login
+		? module?.owner?.login
+		: __( 'Another admin', 'google-site-kit' );
```

Comment:
We should format the fallback in a way that it couldn't be used as an actual username.

```suggestion
		: __( '(Another admin)', 'google-site-kit' );
```